help/grandorgue.xml

```diff
@@ -9896,30 +9896,45 @@ rank size), eg. 2 2/3 =&gt; 64 / (2 2/3) = 24.
      </para>
           </listitem>
         </varlistentry>
-        <varlistentry>
-          <term>Pipe999MIDIKeyNumber</term>
-          <listitem>
-            <para>
+  <varlistentry>
+    <term>Pipe999MIDIKeyNumber</term>
+    <listitem>
+      <para>
 (integer -1 - 127, default: -1) If -1, use pitch information from the
 Pipe999 sample, else override the information in the sample with this
-MIDI note number (Pipe999PitchCorrection is used for specifying the
-fraction). Specifying the MIDI note number also reset the pitch
-fraction in the sample to 0.
-     </para>
-          </listitem>
-        </varlistentry>
-        <varlistentry>
-          <term>Pipe999PitchCorrection</term>
-          <listitem>
-            <para>
-	      (float -1800 - 1800, default: 0) Correction factor in cent for the
-	      pitch specified in the sample in addition to PitchCorrection of
-	      the whole organ, of the windchest and of the rank. This setting is
-	      used for retuning to other temperaments.
-     </para>
-          </listitem>
-        </varlistentry>
-        <varlistentry>
+MIDI note number (Pipe999MIDIPitchFraction is used for specifying the
+fraction). Specifying the MIDI note number also resets the pitch
+fraction in the sample to 0 unless Pipe999MIDIPitchFraction is also specified.
+      </para>
+    </listitem>
+  </varlistentry>
+  <varlistentry>
+    <term>Pipe999MIDIPitchFraction</term>
+    <listitem>
+      <para>
+	(float 0 - 100, default: -1) If -1, use pitch information from the
+	Pipe999 sample, else override the information in the sample with this
+	value. Describes the actual existing pitch fraction in cent over the
+	normal (440hz equal) tone of the midi key specified in the sample or in
+	the Pipe999MIDIKeyNumber. This setting (together with
+	Pipe999MIDIKeyNumber) is used for (perhaps when is a better word than
```

Comment:
Sorry, I was unclear here. I meant:

is used when retuning to other temperaments.

Reply:
Fixed